client/src/languages/javascript.ts

```diff
@@ -85,39 +99,49 @@ class JavascriptBlockKind implements Langs.Block {
       resolve({code: node, exports: dependencies});
     });
   }
-
-  type JavascriptEvent = {}
-  
-  type JavascriptState = {
-    id: number
-    block: JavascriptBlockKind
-  }
   
   const javascriptEditor : Langs.Editor<JavascriptState, JavascriptEvent> = {
     initialize: (id:number, block:Langs.Block) => {  
-      return { id: id, block: <JavascriptBlockKind>block}
+      return { id: id, block: <JavascriptBlockKind>block, tabID:0}
     },
   
-    update: (state:JavascriptState, event:JavascriptEvent) => state,
+    update: (state:JavascriptState, event:JavascriptEvent) => {
+      switch(event.kind) {
+				case 'switchtab':
```

Comment:
There are a few more tabs left in this file.

Reply:
Removed in commit https://github.com/wrattler/wrattler/commit/baf798530a21ccb8198bd9930f199f68f034b553